linebot/liff.go

```diff
@@ -195,7 +204,7 @@ func (call *DeleteLIFFCall) WithContext(ctx context.Context) *DeleteLIFFCall {
 
 // Do method
 func (call *DeleteLIFFCall) Do() (*BasicResponse, error) {
-	endpoint := fmt.Sprintf(APIEndpointDeleteLIFFAPP, call.LIFFID)
+	endpoint := fmt.Sprintf(APIEndpointDeleteLIFFAPP, call.liffID)
```

Comment:
I also want to rename `APIEndpoint*LIFFAPP` constants to `APIEndpoint*LIFFApps`.

Reply:
Ah, I agree. I'll change them, thanks!